src/anm/animation/element.js

```diff
@@ -805,14 +813,13 @@ Element.prototype.render = function(ctx) {
             this.transform(bctx);
             this.painters(bctx);
             this.each(function(child) {
-                child.tick(dt);
                 child.render(bctx, dt);
             });
 
+            // FIXME: this should be performed one time for all masked elements!
             mask.transform(mctx);
```

Comment:
@skavish I am uncertain in this part. The matrix is calculated once, but current bounds affect the mask canvas size. Is it safe to move it to mask render cycle?